judge/judgedaemon.main.php

```diff
@@ -334,7 +334,10 @@ function fetch_executable_internal(
     $execrunpath     = $execbuilddir . '/run';
     $execrunjurypath = $execbuilddir . '/runjury';
     if (!is_dir($execdir) || !file_exists($execdeploypath)) {
-        system('rm -rf ' . dj_escapeshellarg($execdir) . ' ' . dj_escapeshellarg($execbuilddir));
+        system('rm -rf ' . dj_escapeshellarg($execdir) . ' ' . dj_escapeshellarg($execbuilddir), $retval);
+        if ($retval !== 0) {
+            error("Deleting '$execdir' or '$execbuilddir' was unsuccessful.");
```

Comment:
Can we create an internal error disabling this judgehost instead?

Reply:
I assume you also want it for the `mkdir` in the next step? I've added that now.